great_expectations/core/validation_definition.py

```diff
@@ -241,9 +241,10 @@ def _decode_data(cls, data_dict: dict) -> BatchDefinition:
         return batch_definition
 
     @public_api
-    def run(
+    def run(  # noqa: PLR0913
         self,
         *,
+        checkpoint_name: Optional[str] = None,
```

Comment:
Do we need to do a minor version bump for changing the public api or is it ok to do in a patch release because it is optional?